requirements.txt

```diff
@@ -4,4 +4,5 @@ nose==1.3.3
 requests==2.10.0
 celery==3.1.18
 newrelic
-git+https://github.com/yro/chunkey.git
+# this way is required when installing using `pip install requirements.txt` so that `encode_profiles.json` is accessible
+-e git+https://github.com/yro/chunkey.git@14f9c330d4ba5a4290dd0d17429b142618bc2c9f#egg=chunkey
```

Comment:
@muhammad-ammar I've filed https://openedx.atlassian.net/browse/EDUCATOR-2279 for this to go away in future, may you please include its number in the relevant comment so that it can be easily trackable in future.